Names.md

```diff
@@ -24,8 +24,8 @@ This is a document holds the team members of the different sub teams in the A2SV
 
 
 ### Web Team
-* **Full-Time**
-  * Full Name
+* **Full-Time** 
+  * Lensa Billion DTL
```

Comment:
Sorry, my mistake but we have decided to not include the project in the names so please remove that